### DISCUSSION TITLE
Pluto to Lua bytecode: versions, architecture

### DISCUSSION THREAD

**Discussion by CertHaramas** [PlutoLang/Pluto#894] [General]
Hello,
Pluto's features look so great so I'm thiking (againg) about advancing to Lua 5.4.
But the problem is - as usually - other binary dependencies.
So the question:
- Could Pluto (Windows binary) compilers be provided to produce both Lua 32b and 64b bytecode?
- Could Pluto compilers to produce Lua 5.1 bytecode be provided?
(Problem now to obtain/build some libs for Lua 5.4, namely: wx (WxWidgets to Lua), xls (Excel .xls reader/writer.)
TIA.

**Comment by Sainan** [comment]
Pluto is exclusively based on Lua 5.4 right now. We have no plans of providing bytecode generation that is backward compatible. We are more focused on loading bytecode of older versions to faciliate a smooth transition for integrators.

You are free to compile plutoc, targeting 32-bit x86, if you require this. We don't provide 32-bit builds on Github because of lack of perceived value; given that 32-bit has long been on the way out, and its market share is now practically a margin of error.
